stable/redis-ha/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+Redis cluster can be accessed via port 6379 on the following DNS name from within your cluster:
+redis-svc.{{ .Release.Namespace }}.svc.cluster.local
+
+To connect to your Redis server:
+
+1. Run a Redis pod that you can use as a client:
+
+   kubectl exec -it redis-master-0 bash
+
+2. Connect using the Redis CLI:
+
+  redis-cli -h redis-svc.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
Please interpolate values in `NOTES.txt`. The original Redis chart is a good example: https://github.com/kubernetes/charts/blob/master/stable/redis/templates/NOTES.txt